.gitignore

```diff
@@ -116,3 +116,4 @@ example.sites.php
 default.services.yml
 default.settings.php
 settings.php
+web/themes/custom/emulsify
```

Comment:
We don't want to ignore the generated theme. If this stays, then when someone pushed up the code to their own repo the custom theme will not be present.

Reply:
This is in the section labeled `# Remove me after sous install`